bin/build

```diff
@@ -77,6 +83,8 @@ NPM_VERSION=$(get-version --engine npm $PACKAGE_PATH 2>/dev/null) || true
 
 if [[ 'v$NODE_VERSION' != $(node -v) ]]; then
   nvm install $NODE_VERSION > /dev/null 2>&1
+  nvm use $NODE_VERSION
```

Comment:
Maybe a message saying `Rebuilding node modules` since it may take some time depending on how many dependencies a project has.